src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/SearchResultPackagesConverter.cs

```diff
@@ -31,13 +41,16 @@ public override void Write(Utf8JsonWriter writer, IPackageSearchMetadata value,
             writer.WriteStartObject();
             writer.WriteString(JsonProperties.PackageId, value.Identity.Id);
 
-            if (_exactMatch)
-            {
-                writer.WriteString(JsonProperties.Version, value.Identity.Version.ToNormalizedString());
-            }
-            else
+            if (value.Identity.Version is not null)
```

Comment:
This can go away and use the call to `WriteStringIsNotNullOrWhiteSpace` instead:
```c#
WriteStringIsNotNullOrWhiteSpace(writer, JsonProperties.Version, value.Identity.Version?.ToNormalizedString());
```